### PR TITLE
Update ADOPTERS.md

### PR DESCRIPTION
**Pull request by porsche-rishisaxena**
I have updated the Adopters file by adding Porsche AG as one of the users. Please review the content.

Note: Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ This really helps the project to gain momentum and credibility. It's a small con
...
-| [HERE Technologies](https://here.com/) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all its products, as well as to manage licensing risks. |
+| [HERE Technologies](https://here.com/) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all its products, as well as to manage licensing risks. | [Porsche AG](https://www.porsche.com/) | Open_Source_Office@porsche.de | Porsche AG started using oss review toolkit (ort) in the year 2020 where we utilize ORT as a core service for #FOSS Compliance and #FOSS Vulnerability. We are active participants in the ORT community to understand, learn and collaborate on the roadmap defined within the community. Our approach is to run ORT as a service decentrally by decoupling logically Analyzer and Scanner where the Product Development Team takes the responsibility for running the analyzer and the open source program office takes the responsibility for running the scanner, curation, and report consolidation for auditing.
```
Comment (sschuberth) [inline ADOPTERS.md:10]: Please start a new line with

    | [Porsche AG](https://www.porsche.com/)

to fix the formatting.

Comment (sschuberth) [reply ADOPTERS.md:10]: Please just say "ORT" instead of "oss review toolkit (ort)" as readers probably know the acronym.

Comment (sschuberth) [reply ADOPTERS.md:10]: Hashtags like "#FOSS" are not interpreted here, so just say "FOSS".

Comment (sschuberth) [reply ADOPTERS.md:10]: While true, I'd recommend to just drop the "We are active participants in the ORT community to understand, learn and collaborate on the roadmap defined within the community." sentence as this is more about how you're adopting ORT as a tool, and it makes the text a bit shorter.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Comment by sschuberth** [comment]
Also the commit message needs to be amended to adhere to conventional commits. I'd recommend

   docs(ADOPTERS): Add Porsche AG as an adopter of ORT

File: ADOPTERS.md [hunk 2]
```diff
@@ -7,5 +7,8 @@ This really helps the project to gain momentum and credibility. It's a small con
 |--------------|---------|--------------------|
 | [Alliander](https://www.alliander.com/en/open-source/) | ospo@alliander.com | Alliander is mainly interested in the security side of the software supply chain. Alliander uses multiple tools and has evaluated even more, but currently ORT is unbeaten. ORT provides accurate and destailed scan results and allows us to define a through policy to use in evaluation. We are currently working to adopt ORT for thorough scans in addition to other tools. We're working with the community to get ORT running in GitHub Actions. |
 | [Deutsche Telekom AG](https://www.telekom.de/) | daniel.eder@magenta.at | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our [ORT fork](https://github.com/telekom/ort). |
-| [HERE Technologies](https://here.com/) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all its products, as well as to manage licensing risks. |
+| [HERE Technologies](https://here.com/) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all its products, as well as to manage licensing risks.
+|
```
Comment (sschuberth) [inline ADOPTERS.md:11]: This pipe symbol needs to stay at the end of line 10.

File: ADOPTERS.md [hunk 3]
```diff
@@ -7,5 +7,8 @@ This really helps the project to gain momentum and credibility. It's a small con
...
+| [Porsche AG](https://www.porsche.com/) | Open_Source_Office@porsche.de | Porsche AG started using ORT in the year 2020 where we utilize ORT as a core service for FOSS Compliance and FOSS Vulnerability.Our approach is to run ORT as a service decentrally by decoupling logically Analyzer and Scanner where the Product Development Team takes the responsibility for running the analyzer and the open source program office takes the responsibility for running the scanner, curation, and report consolidation for auditing.
+|
```
Comment (sschuberth) [inline ADOPTERS.md:13]: This pipe symbol needs to be put at the end of line 12 (separated with a single space from the `.` at the end of the preceding sentence, like it was originally in line 10).

**Review by sschuberth — CHANGES_REQUESTED** [review]
There are still some minor formatting issues, and the two commits should be squash into one. @porsche-rishisaxena, if you want I can take over the PR and get this done for you.

**Comment by porsche-rishisaxena** [comment]
> There are still some minor formatting issues, and the two commits should be squash into one. @porsche-rishisaxena, if you want I can take over the PR and get this done for you.

Hi @sschuberth. Yes sure please feel free to adjust by taking over this PR. Thank you so much.

**Comment by sschuberth** [comment]
Superseded by https://github.com/oss-review-toolkit/ort/pull/6926.

**Pull request closed without merging**